root/static/scripts/release-editor/fields.js

```diff
@@ -424,6 +424,11 @@ class Medium {
             return !self.canHaveDiscID() && (self.hasExistingTocs() || hasPregap() || hasDataTracks());
         });
 
+        this.hasTooEarlyFormat = ko.computed(function () {
+            const formatDate = MB.formatDates[self.formatID()];
+            return !!self.formatID() && !!self.release.earliestYear() && formatDate && (self.release.earliestYear() < formatDate);
```

Comment:
You can avoid checking `self.formatID()` there again since `formatDate` will definitely be undefined if the format was, and perhaps wrap the whole thing in `!!` for simplicity (one of the terms, `formatDate`, was missing a `!!`). `showErrorRightAway` works fine if the return value isn't boolean anyway, but.

```suggestion
            const formatDate = MB.formatDates[self.formatID()];
            return !!(formatDate && self.release.earliestYear() && self.release.earliestYear() < formatDate);
```